tools/functional-tester/etcd-tester/main.go

```diff
@@ -24,11 +24,13 @@ import (
 func main() {
 	endpointStr := flag.String("agent-endpoints", ":9027", "")
 	datadir := flag.String("data-dir", "agent.etcd", "")
+	stressKeySize := flag.Int("stress-key-size", 100, "")
```

Comment:
we need to add help info to the flag before the blog post. "stress-key-size is the size of each key written into etcd"